### ISSUE TITLE
ERROR: Cannot load buffer. 	[string "LuaMacros script"]:96: '}' expected (to close '{' at line 43) near '['

### ISSUE THREAD

**Issue by SlashTV** [TaranVH/2nd-keyboard#47]
So what exactly does this mean? I'm a noob with programming and lua so I have no idea what to do... My Keyboard an PC is set to German btw. I don't know if that is important.
Please help me :(

**Comment by TaranVH** [comment]
it could mean many, many things.
You do need to be somewhat familiar with basic programming/scripting before you attempt something at this level.
Start here:
https://autohotkey.com/docs/Tutorial.htm
then maybe check out some stuff from www.code.org

**Comment by SlashTV** [comment]
Yeah I have already done some Auto Hotkeys and can also program some C++ so I'm actually not an "absolute beginner" but never coded Lua. So, I also tried to run the script with my first Keyboard set to the macros keyboard but it didn't work either. I did change the path for the keypressed txt and did anything exactly like in the tutorial.  I didn't change any of the code (besides the path as I said) and now I am basically out of ideas for a solution. I would be super happy if it would work because I think it's such a cool thing for video editing but it seems like I have to give up on it for now.
Any ideas?
Slash
